src/main/java/frc/robot/commands/StateMachine.java

```diff
@@ -0,0 +1,76 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import frc.robot.subsystems.Indexer.Indexer;
+import frc.robot.subsystems.Indexer.IndexerState;
+import frc.robot.subsystems.Intake.Intake;
+import frc.robot.subsystems.Intake.IntakeState;
+import frc.robot.subsystems.Shooter.Shooter;
+import frc.robot.subsystems.Shooter.ShooterState;
+
+public class StateMachine extends Command {
+    public enum RobotState {
+        IDLE(IntakeState.IDLE, ShooterState.IDLE, IndexerState.IDLE),
+        INTAKE(IntakeState.INTAKE, ShooterState.IDLE, IndexerState.INDEX),
+        REVERSE(IntakeState.REVERSE, ShooterState.REVERSE, IndexerState.REVERSE),
+        SHOOT(IntakeState.IDLE, ShooterState.SHOOT, IndexerState.IDLE);
```

Comment:
add a RobotState that Indexs which does the same thing as the Intake state without the Intake running